src/Nethermind/Nethermind.Blockchain/Producers/TxPoolTxSource.cs

```diff
@@ -160,11 +167,20 @@ bool HasEnoughFounds(IDictionary<Address, UInt256> balances, Transaction transac
                     continue;
                 }
 
-                if (!HasEnoughFounds(remainingBalance, tx))
+                // ToDo
+                // if (!HasCorrectFeeCap(tx, isEip1559Enabled, baseFee))
```

Comment:
is this TODO still a TODO?

Reply:
use TODO: then it is recognized by the IDE

Reply:
cosmetic - I didn't remove ToDo :)